Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Leonardo DaBepis#4869 on discord / @leonardo-dabepis on Tumblr",
```

Comment:
Did you draw this or did Phil?

Reply:
Leo created the original jacket, I merely edited them to match the color schemes

Reply:
you need to credit yourself for the edit